torchaudio/prototype/ctc_decoder/ctc_decoder.py

```diff
@@ -26,10 +26,12 @@ class Hypothesis(NamedTuple):
     :ivar torch.LongTensor tokens: Predicted sequence of token IDs
     :ivar List[str] words: List of predicted words
     :ivar float score: Score corresponding to hypothesis
+    :ivar torch.IntTensor timestpeps
```

Comment:
Could you mention the shape of the tensor here?
(same goes to `tokens` as well, though strictly speaking it's not this PR's scope)

Reply:
thoughts on how to describe the shape? they are the post-processed tokens and corresponding timesteps list, where blanks and duplicates have already been filtered and so it's shape is different from emission length

ex/ `['|', 'f', 'o', 'o', '|', 'b', 'a', 'r', '|']`

Reply:
Something like "shape `(L, )`, where `L` is the length of output label sequence."?